_posts/2021-08-22-release-2.0.0.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 2.0.0 of MDAnalysis!
+
+This is a new major version, includes major API changes and concludes our current [roadmap][].
+
+This version supports Python 3.6 through to 3.9 on Windows (32- and 64-bit), MacOS, and Linux. It also offers preliminary support for ppc64le and ARM64, but does not currently support Apple M1 processors.
+
+## Upgrading to MDAnalysis version 2.0.0
+
+To install with `conda` from the [conda-forge channel][] run
+
+```bash
+conda update -c conda-forge mdanalysis
+```
+
+To install from [PyPi][] with `pip` run
+
+```bash
+pip install --upgrade MDAnalysis
+```
+
+For more help with installation see the [installation instructions in the User Guide][].
+
+## Notable new additions
+
+There were many awesome contributions to MDAnalysis for version 2.0.0. Here we list a few notable example of these. For more information please see the [CHANGELOG][].
+
+### RDKit converter
+
+[Google Summer of Code][] student @cbouy implemented a new [RDKit converter][] which allows for seamless conversions from RDKit to MDAnalysis structures.
+
+It currently accurately converts 90% of chemical space (based on [ChEMBL27][]), but future changes will improve this.
+
+### OpenMM converter
+
+@ahy3nz implemented an [OpenMM converter][] which can read OpenMM objects directly into MDAnalysis.
+
+### Universe serialization
+
+[Google Summer of Code][] student @yuxuanzhuang implemented a means to serialize MDAnalysis.Universe objects, paving the way for [parallel analysis][].
+
+### Mean Square Displacement
+
+@hmacdope implemented a new [Mean Square Displacement][] analysis class.
+
+### Results class
+
+@PicoCentauri implemented a [Results class][] to store results from analysis classes.
+
+### H5MD support
+
+**NSF REU** student @edisj implemented both a [reader and writer for the H5MD format][].
+
+### Bond-Angle-Torsion coordinates converter
```

Comment:
I wouldn't call BAT a converter because this muddles the waters. Let's only call "converter" the things in our converter framework. Perhaps just "analysis"?

Reply:
I'm not convinced we should call it an "analysis" method to be honest - I should raise an issue about this at some point, but I think it really should have been an MDAnalysis.lib component.

Reply:
fair enough, something for the future and 3.0; I am happy with your change